lib/board.rb

```diff
@@ -1,11 +1,106 @@
 class Board
 
+  attr_reader :size, :grid
+
+  FIRST_ELEMENT = 0
+  NUMBER_ONE = 1
+  NUMBER_TWO = 2
```

Comment:
I see how you have used these constants to avoid having "magic numbers" in your code, nice one for making this more clear.

However, where you have used it in mark_board, I think it would be clearer to call a method that gets the index position of the cell.

e.g. `@grid[index_position(number)] = mark`

Similarly it could be used in `create_grid`

e.g. `(0..index_position(@size)).to_a`

In fact! I think you can create a non-inclusive range in ruby..So!

You can simplify it even more with `(0...@size).to_a`
A non inclusive range has three dots instead of two

Reply:
@DaisyMolving created the `index_position()` method but for the `create grid()`, I wouldn't be able to use a non inclusive range unless I add `1` to `max_number_of_cells`